carma_wm_ros2/include/carma_wm_ros2/collision_detection.hpp

```diff
@@ -42,10 +42,12 @@ namespace carma_wm {
         typedef boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian> point_t;
         typedef boost::geometry::model::polygon<point_t> polygon_t;
 
+        //TODO: verify correctness of descriptions for each of these elements
```

Comment:
Is this todo still open?

Reply:
Yeah both of these TODOs are still open left from John's compile warnings PR